js-api-doc/README.md

```diff
@@ -112,3 +120,38 @@ of Sass code by passing in a {@link LegacyStringOptions}.
     }
   });
   ```
+
+## Speed
+
+While multiple factors go into how long Sass compilations take, there are
+general speed trends that can help you minimize your compilation time.
+
+### With the `sass` package
+
+With the `sass` package, the synchronous calls will be faster than asynchronous
+calls due to the overhead of making the entire evaluation process asynchronous.
+While the {@link Compiler} and {@link AsyncCompiler} class are available, they
+aren't faster than than the module level compilation methods when using `sass`.
+
+### With the `sass-embedded` package
+
+The `sass-embedded` package provides significant speed improvements in certain
+situations, and is generally faster than `sass` for large compilations. When
```

Comment:
I'd say "large or frequent compilations", since the shared resources API now makes it much faster to compile many small Sass files with the embedded host.

Reply:
Fixed in c8b1198eeb33347f8c5df28e6a5ff798d5375c09